src/protocols/aeternity/components/TransferReview.vue

```diff
@@ -148,6 +149,11 @@ export default defineComponent({
       updateMultisigAccounts,
     } = useMultisigAccounts({ store });
     const { getTippingContracts } = useTippingContracts({ store });
+    const {
+      createOrChangeAllowance,
+      burnTriggerPoS,
+      transferToken: fungibleTokenTransfer,
```

Comment:
I think there is no need now to rename the method as `transferToken` is fully understandable and does not exist in the scope.